packages/atomic/src/utils/initialization-utils.spec.ts

```diff
@@ -15,6 +15,11 @@ import {
   MissingInterfaceParentError,
 } from './initialization-utils';
 
+/* eslint-disable prettier/prettier */
+// TODO: get rid of the mock once no longer running stencil tests
```

Comment:
No need to add jira ticket as this will eventually have to be replaced during lit migration